lnbits/static/js/event-reactions.js

```diff
@@ -1,16 +1,30 @@
-function eventReactionWebocket(event_id, event_function) {
+function eventReactionWebocket(event_id) {
   localUrl = ''
+  reaction = localStorage.getItem('lnbits.reactions')
+  if (reaction == null || reaction == 'None' || reaction == undefined) {
```

Comment:
```suggestion
  if (!reaction) {
```

I don't see how `reaction` can be `None`.

Reply:
Its an option in the account section, if you don't want reactions

Reply:
nitpick:
```suggestion
  if (!reaction || reaction === 'None') {
```